example/package.json

```diff
@@ -17,10 +17,10 @@
     "@types/react": "file:../node_modules/@types/react",
     "@types/react-dom": "file:../node_modules/@types/react-dom",
     "axios": "0.24.0",
-    "react-router-dom": "6.2.1",
-    "react": "file:../node_modules/react",
     "bootstrap": "file:../node_modules/bootstrap",
+    "react": "^17.0.2",
```

Comment:
We need to preserve relative imports to not have more packages to keep up to date